src/core/stream.c

```diff
@@ -659,6 +661,38 @@ QuicStreamParamSet(
         break;
     }
 
+   case QUIC_PARAM_STREAM_RELIABLE_OFFSET_SEND:
```

Comment:
Since the peer won't be setting the receive, only send, let's remove `_SEND` from the name here.

Reply:
Done.